tests/scancode/test_ignore_files.py

```diff
@@ -74,3 +81,45 @@ def test_ignore_glob_file(self):
             {}
         )
         assert is_ignored(*test)
+
+    def test_resource_paths_1(self):
```

Comment:
Rather than _1 could you use a prefix in your test name that explains what you are testing?